common/src/main/java/com/microsoft/identity/common/internal/broker/BrokerData.java

```diff
@@ -66,6 +69,16 @@
             BROKER_HOST_APP_SIGNATURE
     );
 
+    public static final BrokerData INTUNE_NGMS_AGENT_DEBUG = new BrokerData(
+            INTUNE_NGMS_AGENT_APP_PACKAGE_NAME,
+            INTUNE_NGMS_AGENT_APP_DEBUG_SIGNATURE
+    );
+
+    public static final BrokerData INTUNE_NGMS_AGENT_PROD = new BrokerData(
```

Comment:
What we really want to vend here is essentially an object that contains a name and a collection of valid signatures for that name.  If we did that, everything would fall nicely out, I think.